addOns/bruteforce/src/main/java/com/sittinglittleduck/DirBuster/HTMLparse.java

```diff
@@ -130,16 +130,15 @@ public void run() {
                                 }
 
                             } catch (MalformedURLException e) {
-                                // System.out.println("Man thats a bad url!");
+                                LOG.error("Bad URL", e);
                             }
                         }
 
                         try {
-
                             Thread.sleep(100);
                         } catch (InterruptedException ex) {
+                            LOG.error(ex);
```

Comment:
I'm not sure we want to log this (and other InterruptedExceptions) as an error.
In ZAP we tend to use LOG.error(..) for things that imply that we need to fix something in the code.
Ideally things that are user errors should be reported via the GUI, but thats harder for packages like this which we've inherited.
For this sort of case I'd say we should log errors if either there could be a bug in ZAP or if we want to inform the user and theres no easy way in the GUI.
If its unlikely to be noticed unless the user is actively investigating a problem then I'd recommend LOG.debug(..) but in the case of InterupdatedException I dont thin theres anything either we or the user would do, so I'd recommend not logging anything.

Reply:
Thanks for the comment. I will fix it.